### PR TITLE
Enhanced monitoring

### PR DESCRIPTION
**Pull request by waynz0r**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    |yes
| API breaks?     | no|yes
| Deprecations?   | no|yes
| Related tickets | 
| License         | Apache 2.0

### What's in this PR?

Use latest Backyards chart which has enhanced monitoring dashboards.

### CODE REVIEW

File: internal/cli/cmd/chartvalues.go [hunk 1]
```diff
@@ -280,4 +296,6 @@ func (values *Values) SetDefaults(releaseName, istioNamespace string) {
 
 	values.Auth.Mode = anonymous
 	values.Impersonation.Enabled = false
+
+	values.KubeStateMetrics.Enabled = true
```
Comment (Laci21) [inline internal/cli/cmd/chartvalues.go]: Is there a point setting it to false in the values.yaml when it is set to true here anyway?
https://github.com/banzaicloud/backyards-cli/pull/119/files#diff-7e4478a97587d1701d33a8a22fee0daaR113

File: internal/cli/cmd/uninstall.go [hunk 2]
```diff
@@ -212,3 +224,32 @@ func (c *uninstallCommand) runSubcommands(cli cli.CLI, options *UninstallOptions
...
+	if err != nil {
+		return err
+	}
+
+	m, err := nodeexporter.NewNodeExporterManager(resourcemanager.New(client, c.cli.LabelManager()), c.cli.GetPersistentConfig().Namespace())
+	if err != nil {
+		return err
+	}
+	if options.dumpResources {
+		yaml, err := m.Install().YAML()
```
Comment (Laci21) [inline internal/cli/cmd/uninstall.go]: Shouldn't this be?
```suggestion
		yaml, err := m.Uninstall().YAML()
```

File: pkg/nodeexporter/nodeexporter.go [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+	values.SetDefaults()
+
+	rawValues, err := yaml.Marshal(values)
+	if err != nil {
+		return nil, errors.WrapIf(err, "could not marshal yaml values")
+	}
+
+	objects, err := helm.Render(nodeexporter.Chart, string(rawValues), helm.ReleaseOptions{
+		Name:      "backyards",
```
Comment (Laci21) [inline pkg/nodeexporter/nodeexporter.go]:
```suggestion
		Name:      "prometheus-node-exporter",
```

File: internal/cli/cmd/chartvalues.go (hunk 1 above)
Comment (waynz0r) [reply internal/cli/cmd/chartvalues.go]: No harm setting it here, just to be sure. :)

**Pull request merged**